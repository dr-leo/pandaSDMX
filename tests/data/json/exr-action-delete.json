{
    "header": {
        "id": "62b5f19d-f1c9-495d-8446-a3661ed24753",
        "prepared": "2012-11-29T08:40:26Z",
        "sender": {
            "id": "ECB",
            "name": "European Central Bank"
        }
    },
    "structure": {
        "links": [
          {
            "href": "https://sdw-wsrest.ecb.europa.eu/service/dataflow/ECB/EXR/1.0",
            "rel": "dataflow"
          }
        ],
        "dimensions": {
            "dataSet": [
                {
                    "id": "FREQ",
                    "name": "Frequency",
                    "role": "FREQ",
                    "keyPosition": 0,
                    "values": [
                        {
                            "id": "D",
                            "name": "Daily"
                        }
                    ]
                }, {
                    "id": "CURRENCY_DENOM",
                    "name": "Currency denominator",
                    "keyPosition": 2,
                    "values": [
                        {
                            "id": "EUR",
                            "name": "Euro"
                        }
                    ]
                }, {
                    "id": "EXR_TYPE",
                    "name": "Exchange rate type",
                    "keyPosition": 3,
                    "values": [
                        {
                            "id": "SP00",
                            "name": "Spot rate"
                        }
                    ]
                }, {
                    "id": "EXR_SUFFIX",
                    "name": "Series variation - EXR context",
                    "keyPosition": 4,
                    "values": [
                        {
                            "id": "A",
                            "name": "Average or standardised measure for given frequency"
                        }
                    ]
                }
            ],
            "series": [
                {
                    "id": "TIME_PERIOD",
                    "name": "Time period or range",
                    "role": "TIME_PERIOD",
                    "values": [
                        {
                            "id": "2013-01-18",
                            "name": "2013-01-18",
                            "start": "2013-01-18T00:00:00.000Z",
                            "end": "2013-01-18T23:59:59.999Z"
                        }, {
                            "id": "2013-01-21",
                            "name": "2013-01-21",
                            "start": "2013-01-21T00:00:00.000Z",
                            "end": "2013-01-21T23:59:59.999Z"
                        }
                    ]
                }
            ],
            "observation": [
                {
                    "id": "CURRENCY",
                    "name": "Currency",
                    "keyPosition": 1,
                    "values": [
                        {
                            "id": "NZD",
                            "name": "New Zealand dollar"
                        }, {
                            "id": "RUB",
                            "name": "Russian rouble"
                        }
                    ]
                }
            ]
        },
        "attributes": {
            "dataSet": [],
            "series": [],
            "observation": [
                {
                    "id": "TITLE",
                    "name": "Series title",
                    "role": "TITLE",
                    "values": [
                        {
                            "name": "New Zealand dollar (NZD)"
                        }, {
                            "name": "Russian rouble (RUB)"
                        }
                    ]
                }, {
                    "id": "OBS_STATUS",
                    "name": "Observation status",
                    "values": [
                        {
                            "id": "A",
                            "name": "Normal value"
                        }
                    ]
                }
            ]
        }
    },
    "dataSets": [
        {
            "action": "Replace",
            "validFrom": "2012-03-20T15:00:16.000+01:00",
            "series": {
                "0": {
                    "attributes": [],
                    "observations": {
                        "1": [40.3426, 1, 0]
                    }
                },
                "1": {
                    "attributes": [],
                    "observations": {
                        "1": [40.3000, 1, 0]
                    }
                }
            }
        }, {
            "action": "Delete",
            "validTo": "2012-03-20T10:07:58.000+01:00",
            "series": {
                "0": {
                    "observations": {
                        "0": []
                    }
                }
            }
        }
    ]
}
