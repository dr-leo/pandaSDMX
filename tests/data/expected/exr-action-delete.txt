FREQ                    D          D
TIME_PERIOD    2013-01-18 2013-01-21
CURRENCY_DENOM        EUR        EUR
EXR_TYPE             SP00       SP00
EXR_SUFFIX              A          A
CURRENCY
RUB               40.3426       40.3
